content/learn-more/_index.html

```diff
@@ -0,0 +1,169 @@
+          </div>
+        </div>
+        <div class="column is-one-third">
+          <h2><i class="fa-light fa-layer-plus"></i> Scalable </h2>
+          <p> RAPIDS delivers impressive acceleration from on a single GPU desktop system all the way to MNMG (multi-node multi-GPU) configurations on an expansive range of <a href="https://www.nvidia.com/en-us/deep-learning-ai/solutions/data-science/" target="_blank">NVIDIA hardware</a>.  </p>
+          <div class="content">
+            <ul> 
+              <li>Impressive single GPU performance out of the box</li>
+              <li>MNMG with <a href="https://docs.dask.org/en/stable/gpu.html" taget="_blank"> RAPIDS + Dask</a> and <a href="https://nvidia.github.io/spark-rapids/" targe="_blank">RAPIDS + Spark</a></li>
+              <li>HPC with <a href="https://docs.rapids.ai/deployment/stable/hpc.html" target="_blank">RAPIDS + Slurm</a></li>
+            </ul>
+          </div>
+        </div>
+        <div class="column is-one-third">
+          <h2><i class="fa-light fa-message-code"></i> Open Source </h2>
+          <p> RAPIDS had its start from the <a href="https://arrow.apache.org/" target="_blank">Apache Arrow</a> and <a href="https://developer.nvidia.com/blog/goai-open-gpu-accelerated-data-analytics/" target="_blank">GoAi</a> projects based on a columnar, in-memory data structure that delivers efficient and fast data interchange with flexibility to support complex data models. With <a href="https://developer.nvidia.com/rapids" target="_blank">NVIDIA backing</a>, it has maintained a focus on open source development (typically Apache 2.0 licensed) and collaboration with a broad community of cutting edge data science projects. </p>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <!-- Use Cases -->
+  <div id="use-cases"></div>
+  <div class="top-slant-container">
+    <div class="top-slant-up has-background-rapids-gray"></div>
+  </div>
+
+  <section class="section has-background-rapids-gray pb-8">
+    <div class="container">
+
+      <h1 class="mb-4">Enterprise Use Cases</h1>
+      <div class="columns">
+        <div class="column is-one-half">
+          <h2><i class="fa-light fa-cart-shopping-fast"></i> Walmart + RAPIDS ML</h2>
+          <p> When you are a global-scale company, precision is everything. Learn how Walmart implemented XGBoost to increase forecast accuracy and potentially save billions of dollars.
+          <a href="https://www.datanami.com/2019/03/22/how-walmart-uses-gpus-for-better-demand-forecasting/" target="_blank">Read More <i class="fa-solid fa-arrow-up-right"></i> </a> </p>
+        </div>
+        <div class="column is-one-half">
+          <h2><i class="fa-light fa-bee"></i> Bumble + RAPIDS ML</h2>
+          <p> Buzzwords is Bumble's open-source GPU-powered topic modelling tool, developed in house and building upon the work found in BERTopic and Top2Vec. Buzzwords uses cuML's UMAP and HDBSCAN algorithms.
```

Comment:
According to a quick search, "modeling" is the US spelling and "modelling" is the UK spelling. Let's use the US form.
```suggestion
          <p> Buzzwords is Bumble's open-source GPU-powered topic modeling tool, developed in house and building upon the work found in BERTopic and Top2Vec. Buzzwords uses cuML's UMAP and HDBSCAN algorithms.
```